config/docs.config.js

```diff
@@ -164,30 +162,6 @@ module.exports = {
     "**/*.spec.jsx",
     "**/ExampleComponent.vue",
   ],
-  webpackConfig: merge(baseConfig, {
-    module: {
-      rules: [
-        {
-          test: /\.(css?|scss|sass)(\?.*)?$/,
-          use: [
-            "style-loader",
-            "css-loader",
-            "sass-loader",
-            {
-              loader: "sass-resources-loader",
-              options: {
-                resources: [
-                  path.join(__dirname, "../src/assets/tokens/tokens.scss"),
-                  path.join(__dirname, "../src/assets/tokens/tokens.map.scss"),
-                  path.join(__dirname, "../src/styles/styles.scss"),
```

Comment:
How are we going to make tokens and SCSS helpers available globally for all components (previously here in webpackConfig)?